src/components/dashboard/FaceVerification/utils/strings.js

```diff
@@ -1 +1 @@
-export const zoomResultSuccessMessage = `You are an\namazing unicorn!`
+export const zoomResultSuccessMessage = `You’re a beautiful\n& unique snowflake!`
```

Comment:
According to the ZoOm i18n examples, an HTML `<br>` tag should be used to add line break to any UI label:

```suggestion
export const zoomResultSuccessMessage = `You’re a beautiful<br>& unique snowflake!`
```

<img width="652" alt="image" src="https://user-images.githubusercontent.com/62208067/89899779-74b46b80-dbeb-11ea-95eb-3245a7a3426d.png">